src/ape/api/accounts.py

```diff
@@ -433,6 +466,109 @@ def get_deployment_address(self, nonce: Optional[int] = None) -> AddressType:
         nonce = self.nonce if nonce is None else nonce
         return ecosystem.get_deployment_address(self.address, nonce)
 
+    @property
+    def delegate(self) -> Optional[ContractInstance]:
+        """
+        Check and see if Account has a "delegate" contract, which is a contract that this account
+        delegates functionality to. This could be from many contexts, such as a Smart Wallet like
+        Safe (https://github.com/ApeWorX/ape-safe) which has a Singleton class it forwards to, or
+        an EOA using a EIP7702-style delegate. Returning `None` means that the account does not
+        have a delegate avaiable, which should be overrided in plugins that support ecosystems
+        where such methods are possible natively (e.g. EIP7702 context), or plugins such as smart
+        wallets which have this functionality. The default implementation is to return `None`.
+
+        Returns:
+            Optional[`:class:~ape.contracts.ContractInstance`]:
+                The contract instance of the delegate contract (if available).
+        """
+        return None
+
+    def set_delegate(self, contract: ContractInstance, **txn_kwargs):
+        """
+        Have the account class override the value of it's `delegate`. For plugins that support this
+        feature, the way they choose to handle it can vary. For example, it could be a call to up-
+        grade itself using some built-in method for a smart wallet (with default txn args) e.g. the
+        Safe smart wallet (https://github.com/ApeWorX/ape-safe), or it could be to use an EIP7702-
+        like feature available on the network to set a delegate for that account. However it a
+        plugin chooses to handle it, the resulting action (if successful) should make sure that the
+        value that `self.delegate` returns is the same as `contract` after it is completed.
```

Comment:
you should use double tics in the docstring to do what you want normal tics to do.